students/Ramkumar_Rajanbabu/Lesson03/Slicing_Lab.py

```diff
@@ -0,0 +1,42 @@
+#Slicing Lab
+
+seq = "this is a string"
+tup = (2, 54, 13, 12, 5, 32)
+
+def exchange_first_last(seq):
+    first = seq[-1] 
+    mid = seq[1:-1]
+    last = seq[0]
+    new_seq = first + mid + last 
+    return new_seq
```

Comment:
this needs to use slicing only - this doesn't work for lists/tuple

Reply:
I'm still working on the slicing lab. Also, I'm still fixing the exchange_first_last since as you mentioned, this doesn't work for lists/tuples. Am I doing the pull request incorrectly? How would I do a pull request after completing an assignment while also starting a new exercise, for example the slicing lab?